### PR TITLE
Add barbican-operator to Makefile

### PR DESCRIPTION
**Pull request by dmendiza**
This PR adds barbican-operator make targets to the install_yamls Makefile.  This PR supersedes #258

### CODE REVIEW

**Review by abays — APPROVED** [review]
/lgtm

File: Makefile [hunk 1]
```diff
@@ -403,11 +414,10 @@ help: ## Display this help.
 	@awk 'BEGIN {FS = ":.*##"; printf "\nUsage:\n  make \033[36m<target>\033[0m\n"} /^[a-zA-Z_0-9-]+:.*?##/ { printf "  \033[36m%-15s\033[0m %s\n", $$1, $$2 } /^##@/ { printf "\n\033[1m%s\033[0m\n", substr($$0, 5) } ' $(MAKEFILE_LIST)
 
 .PHONY: cleanup
-
-cleanup: heat_cleanup horizon_cleanup nova_cleanup octavia_cleanup designate_cleanup neutron_cleanup ovn_cleanup ironic_cleanup cinder_cleanup glance_cleanup placement_cleanup swift_cleanup keystone_cleanup mariadb_cleanup telemetry_cleanup dataplane_cleanup ansibleee_cleanup rabbitmq_cleanup infra_cleanup ## Delete all operators
+cleanup: heat_cleanup horizon_cleanup nova_cleanup octavia_cleanup designate_cleanup neutron_cleanup ovn_cleanup ironic_cleanup cinder_cleanup glance_cleanup placement_cleanup swift_cleanup barbican_cleanup keystone_cleanup mariadb_cleanup telemetry_cleanup dataplane_cleanup ansibleee_cleanup rabbitmq_cleanup infra_cleanup ## Delete all operators
```
Comment (fao89) [inline Makefile:417]: shouldn't `barbican_cleanup` be added here?

Comment (dmendiza) [reply Makefile:417]: `barbican_cleanup` is indeed added there after `swift_cleanup`

Comment (fao89) [reply Makefile:417]: my bad! I didn't see it

**Review by fao89 — APPROVED** [review]

**Pull request merged**